### ISSUE TITLE
Add section to docs on how to use Google Drive and Sheets

### ISSUE THREAD

**Issue by yanokwa** [getodk/docs#748]
https://forum.opendatakit.org/t/invalid-sheet-id-when-submitting-to-google-sheets/13703 reminds me that we have a way for people to go end-to-end with Collect and Aggregate, but it'd be really nice to have some docs written for how to do this with Google Drive.

Add a warning that you [can't use uploaded Excel files](https://forum.opendatakit.org/t/error-400-bad-request-failed-precondition-on-collect-to-google-sheets/19801)

**Comment by yanokwa** [comment]
One comment from users..

>  It would also be good to mention out here that for cloud based docs the sharing settings of the submission sheet should be set to 'Anyone on the Web can Find and Edit' and not 'Public: Anyone with the link can access and edit'.

**Comment by yanokwa** [comment]
Just brainstorming here, but I think there are really two things that need to be documented.
1. You need to create a Google Sheet to get the Sheet URL. That sheet has to have specific permissions.
1. Then that Sheet URL goes into the submission_url of the form (in XLSForm or in Build)

The first, there isn't a great place for it in the current docs. Maybe in tips and best practices in the form building section? Then it can be linked to from the Collect docs on connecting to Google Drive, the Build docs on form properties, and then maybe in a new Form Settings docs that talks about the settings sheet.

**Comment by drinckes** [comment]
The Google Earth team did a good description of setting it up ODK for use with [Drive and Sheets](https://www.google.com/earth/outreach/learn/odk-collect-and-google-drive-integration-to-store-and-manage-your-data/).

**Comment by clint-tseng** [comment]
tried to work on https://github.com/opendatakit/build/issues/204 but it's blocked on this issue still. (just commenting for my own future reference)

**Comment by grzesiek2010** [comment]
We can close as GD integration was removed from Collect.
